### DISCUSSION TITLE
Why are directories missing post-install?

### DISCUSSION THREAD

**Discussion by pmatulis** [mmistakes/minimal-mistakes#3048] [Q&A] [answered]
I installed my site using the GH [instructions](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll) and installed the theme using the [remote theme method](https://mmistakes.github.io/minimal-mistakes/docs/quick-start-guide/#remote-theme-method).

I have this theme running locally and on GH Pages but I see that directories (like `_layouts` and `_includes`) and their contents are missing. Even the `_config.yml` file is impoverished. What is the point of that and what do I need to do in order to have a complete theme? Copy and paste stuff from the theme repository?

This [instruction](https://mmistakes.github.io/minimal-mistakes/docs/installation/) was confusing as I did not need to do it:

> If you plan to host with GitHub Pages be sure to properly setup jekyll-remote-theme as it is required for the theme to work properly.

The output to `gem list` nevertheless contains `jekyll-remote-theme`.

**Comment by mmistakes** [comment, marked as answer]
When you "install" a theme whether as the gem or using `remote_theme` it doesn't add the `_layouts`, `_includes`, `_sass` etc. to your repository directly. Instead it uses those files that are bundled in the theme.

You can certainly copy over those files if you want, but I would advise against that as it defeats the purpose of "installing" the theme. Having copies of those files in your repo will have them override those bundled in the theme. Meaning when you upgrade the theme you need to copy over all of the updated files manually.

If you want "a complete theme" then you'd want to fork this repo. But again, I would advise against that unless you want to maintain those files... making sure they're always update to date with this repo. Forking is more for when you want to take the theme and build on it or change it.

**Reply by pmatulis** [reply]
Thanks for the quick turnaround.

So, effectively speaking, those directories and files still exist somewhere? Does this mean that any configuration takes place only within `_config.yml`, and I'm good to go?

**Reply by mmistakes** [reply]
Yes, that’s correct.
